docs/tutorials/getting_started.ipynb

```diff
@@ -305,7 +304,7 @@
    },
    "outputs": [],
    "source": [
-    "dataset = naip + chesapeake"
+    "dataset = naip & chesapeake"
```

Comment:
I would like to do something like I did in the README where we take the union between ChesapeakeDE and ChesapeakeMD and then take the intersection with NAIP, but the planetary computer STAC API query website thiny seems to be broken so I couldn't use it to find the URL for NAIP tiles over MD. Also, MD is broken (#59) so I'd have to choose a different state, and a lot of those states timeout because the download takes too long. Might have to rework the entire thing to use Landsat 7/8 and CDL like I do in the README.

Reply:
Can we just not use MD?

Reply:
We can auto-download it but we can't auto-extract it. We would have to call to `unzip` which only works on macOS/Linux. Eventually I need to add an optional dep on zipfile-deflate64 and figure out how to tell when it is needed.